libs/dh/message-archive/feature-log-search/src/lib/blobcontent/dh-message-archive-log-search-blob-content.component.scss

```
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.message-id-header {
+  padding-top: 1em;
```

suggestion: consider using one of the predefined CSS variables. `--watt-space-m` has a value of `1rem`.

```suggestion
  padding-top: var(--watt-space-m);
```